drivers/sensor/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ add_subdirectory_ifdef(CONFIG_BME680		bme680)
 add_subdirectory_ifdef(CONFIG_BMG160		bmg160)
 add_subdirectory_ifdef(CONFIG_BMI160		bmi160)
 add_subdirectory_ifdef(CONFIG_BMM150		bmm150)
+add_subdirectory_ifdef(CONFIG_BQ274XX           bq274xx)
```

Comment:
Please replace the spaces with tab(s) to be consistent with the rest of the file

Reply:
Done thanks.